src/store/getters/state-getters.ts

```diff
@@ -237,3 +240,24 @@ export const getMaxStep = (state: any, getters: any): number => {
 export const getValidateSteps = (state: any, getters: any): boolean => {
   return state.stateModel.validateSteps
 }
+
+/** The current state of the certify section. */
+export const getCertifyState = (state: any): CertifyIF => {
+  return state.stateModel.certifyState
+}
+
+/** Business contact email address. */
+export const getBusinessEmail = (state: any, getters: any): string => {
+  return state.stateModel.defineCompanyStep.businessContact.email
+}
+
+/** The Completing Party's email address. */
+export const getCompletingPartyEmail = (state: any): string | null => {
+  const completingParty =
+    state.stateModel.addPeopleAndRoleStep.orgPeople.find(person => {
+      return !!person.roles?.some(role => {
```

Comment:
No need to change, but if the evaluation looks complicated due to `!!...`, you can also do `Boolean(...)`, which can look cleaner.

Reply:
Updated in next commit